common-api/src/main/scala/pl/touk/nussknacker/engine/graph/expression/Expression.scala

```diff
@@ -8,10 +8,12 @@ object Expression {
   object Language {
     val Spel = "spel"
     val SpelTemplate = "spelTemplate"
+    val SqlSpelTemplate = "sqlSpelTemplate"
```

Comment:
Please either: remove this temporary language or keep in this PR only scope of the first, backward compatible step (with old `}#` notation)